app/main.py

```diff
@@ -227,139 +263,197 @@ def enrich_gis_dataframe(gis_dataframe: pd.DataFrame, mrid_dataframe: pd.DataFra
+    # Creating a column with the ETS name
+    gis_dataframe[TEMP_COLUMN] = gis_dataframe[gis_column_name].map(
+        gis_line_name_to_ets_line_name
+    )
 
+    # TODO this seems very wierd looking at it, why remove data??
     # Removing any names that could not be translated to represent a clean dataframe that only contains
     # information that can be looked up with MRID.
     gis_dataframe = gis_dataframe[~gis_dataframe[gis_column_name].isin(bad_line_names)]
 
-    return gis_dataframe
+    dlr_dataframe = mrid_dataframe.join(
+        gis_dataframe.set_index(TEMP_COLUMN),
+        on=ets_column_name,
+        how="inner",
+    )
+
+    # Resetting index
+    dlr_dataframe.reset_index(inplace=True, drop=True)
 
+    return dlr_dataframe
 
-def verify_translated_names_against_ets(translate_gis_name_to_ets: dict[str, str], mrid_dataframe: pd.DataFrame,
-                                        ets_column_name: str):
-    '''
+
+def verify_translated_names_against_ets(
+    translate_gis_name_to_ets: dict[str, str],
+    dataframe: pd.DataFrame,
+    ets_name_column: str,
+    ets_dlr_enabled_column: str,
+):
+    """
     The function checks if there are lines in the ETS dataset which is not represented in the GIS dataset.
-    This is for logging and to be able to see when introducing a new dataset wether the quality stays the same.
+    This is for logging and to be able to see when introducing a new dataset if the data quality stays the same.
 
     Parameters
     ----------
-    gis_line_name_to_ets_line_name: dict[str, str]
+    translate_gis_name_to_ets : dict[str, str]
         The dict containing the translated names from GIS.
-    mrid_dataframe: pd.DataFrame
+    dataframe : pd.DataFrame
         Dataframe with all ETS information in.
-    ETS_COLUMN_NAME: str
+    ets_name_column : str
         The specific column in the MRID dataframe that contains the names of lines.
-
+    ets_dlr_enabled_column : str
+        Name of the column in mrid_dataframe which contains enabled state
     Returns
     -------
     None
-    '''
-    REGEX = r'^[CDE]_'
+    """
+
+    regex = r"^[CDE]_"
 
     # Comparing the converted names from GIS to the list of names in ETS.
-    lines_not_in_gis = list(set(mrid_dataframe[ets_column_name]).difference(list(translate_gis_name_to_ets.values())))
-    # The result is filtered with a regex expression to omit low voltage levels from ETS.
-    missing_lines_in_gis = list(line for line in lines_not_in_gis if re.match(REGEX, line))
+    # omitting low voltages level
+    lines_not_in_gis = list(
+        line
+        for line in (
+            list(
+                set(dataframe[ets_name_column]).difference(
+                    list(translate_gis_name_to_ets.values())
+                )
+            )
+        )
+        if re.match(regex, line)
+    )
+
+    lines_not_in_gis.sort()
+    log.info(f"List of not found lines in GIS data set: {lines_not_in_gis}")
```

Comment:
```suggestion
    log.info(f"List of lines not found in GIS data set: {lines_not_in_gis}")
```
Funny arrangement of words! ;)